src/AspNetCoreModuleV2/CommonLib/hostfxroptions.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#pragma once
+
+class HOSTFXR_OPTIONS
+{
+public:
+    HOSTFXR_OPTIONS(PCWSTR hostFxrLocation, PCWSTR exeLocation, DWORD argc, BSTR argv[]):
+        m_argc(argc),
+        m_argv(argv)
+    {
+        m_hostFxrLocation.Copy(hostFxrLocation);
```

Comment:
@pan-wang will make you add a Populate method as copy returns an HR.

Reply:
Ugh.

Reply:
I don't care as I think the only hr will be E_OUTOFMEM, which I want us to stop avoiding.